src/app/todo/page.tsx

```diff
@@ -0,0 +1,19 @@
+import { getServerSession } from 'next-auth';
+import { redirect } from 'next/navigation';
+import { SignOutButton } from '@/components/SignIn/SignOutButton';
+
+export default async function Page() {
+  const session = await getServerSession();
+
+  if (!session) {
+    redirect('/');
+  }
+
+  return (
+    <>
+      <p>Welcome to the App, {session?.user?.name}!</p>
+      {session?.user?.image && <img src={session?.user?.image} alt="프로필" />}
```

Comment:
```suggestion
      {session.user?.image && <img src={session.user.image} alt="프로필" />}
```
앞쪽에서 유효성 검사를 해주었다면 뒷 부분에서는 체이닝을 걸어줄 필요가 없습니다!
확인해보니 session.user에는 체이닝을 걸지 않아도 괜찮더라구요~